Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Asset/AssetConnection.tsx

```diff
@@ -38,7 +38,7 @@ interface AssetConnection {
     AssetKey: string
 }
 
-function AssetConnectionWindow(props: { Asset: OpenXDA.Types.Asset }): JSX.Element{
+function AssetConnectionWindow(props: { Asset: OpenXDA.Types.Asset, Name: string, ID: number}): JSX.Element{
```

Comment:
```suggestion
function AssetConnectionWindow(props: { Name: string, ID: number}): JSX.Element{
```

actually make this an `interface IProps { ... }`